tasks/src/Reinforcement-learning/about.md

```diff
@@ -0,0 +1,9 @@
+## Use Cases
+
+### Robotics
+
+Reinforcement learning is used in robotics to enable robots to learn from experience and improve their performance over time. For example, a robot might be programmed to clean a floor by starting in one corner and moving in a straight line until it reaches the other corner. If the robot encounters an obstacle, it will learn from this experience and try a different route the next time it cleans the floor.
+
+### Gaming
+
+Reinforcement learning is used in gaming to provide game characters with a set of possible actions and then reward them for selecting the optimal action. This allows game characters to learn from their mistakes and eventually become better at choosing the optimal actions.
```

Comment:
```suggestion
Reinforcement learning is used in video games to provide non-playable characters with a set of possible actions. The trained model then rewards them for selecting the optimal action. This allows non-playable characters to learn from their mistakes and eventually become better at choosing the optimal actions.
```

Reply:
In game industry we don't use RL as replace of NPC (non playable characters) we use FSM (finite state machine), Behavior Tree or GOAP settings.

But I'm agree that's a good explanation so we can keep this way.